packages/libs/react-ui/src/components/Form/RadioGroup/RadioGroup.tsx

```diff
@@ -14,10 +14,11 @@ type Direction = NonNullable<RecipeVariants<typeof groupClass>>['direction'];
 export interface IRadioGroupProps extends AriaRadioGroupProps {
   children: ReactElement[] | ReactElement;
   direction: Direction;
+  info?: string;
+  inverse?: boolean;
```

Comment:
so with inverse the button AND the label text inverses.
button I can understand. But text becomes unreadable. is this desired?
sane goes for checkbox

![image](https://github.com/kadena-community/kadena.js/assets/4015521/10507c55-e483-4ade-9012-ad4ee855ad4a)